lib/rollup.config.js

```diff
@@ -2,11 +2,15 @@ import path from 'path';
 import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 import replace from 'rollup-plugin-replace';
-import typescript from 'rollup-plugin-typescript';
+import typescriptPlugin from 'rollup-plugin-typescript';
+import typescript from 'typescript';
 import { uglify } from 'rollup-plugin-uglify';
```

Comment:
Guys, can we go with babel instead? Or do you miss some not implemented yet features?

Reply:
I don't have any preference.  I don't think there is an issue with babel, I was just trying to get the ts tests done and am bumping into problems with a final build.

Reply:
is there a reference project good to look at for ts + babel + esm rollup?

Reply:
It's babel + rollup usually. Typescript is only for type checking and definitions generating.

Reply:
Is there a good reference project for rollup that outputs esm and cjs with babel?

Reply:
found one https://github.com/rollup/rollup-starter-lib